Python/specialize.c

```diff
@@ -794,6 +794,10 @@ _Py_Specialize_LoadAttr(PyObject *owner, _Py_CODEUNIT *instr, PyObject *name)
             if (!function_check_args(fget, 1, LOAD_ATTR)) {
                 goto fail;
             }
+            if (instr->op.arg & 1) {
```

Comment:
Is this necessary to swap the `NULL` and callable, or just a simplification based on the stats?